compiler/rustc_abi/src/lib.rs

```diff
@@ -28,9 +27,6 @@ pub use layout::LayoutCalculator;
 /// instead of implementing everything in `rustc_middle`.
 pub trait HashStableContext {}
 
-use Integer::*;
-use Primitive::*;
```

Comment:
I don't mind, but just wandering, why this change?

Reply:
Since I was expanding the other globs in this module already I just moved these as well. So really just personal preference (and it makes things easier to reason about to me at least)

Reply:
I can revert this one if wanted

Reply:
I generally like local enum glob imports more, so no need to change anything =)